firmware/quintuna/VC/src/app/app_powerManager.c

```diff
@@ -43,22 +45,39 @@ void app_powerManager_updateConfig(const PowerManagerConfig new_power_manager_co
     power_manager_state = new_power_manager_config;
 }
 
-#define CURRENT_THRESH 0.025f
 static bool STLoadswitch_Status(const ST_LoadSwitch *loadswitch)
 {
     assert(loadswitch->efuse1 != NULL && loadswitch->efuse2 != NULL);
-    if (io_loadswitch_isChannelEnabled(loadswitch->efuse1) &&
-        io_loadswitch_getChannelCurrent(loadswitch->efuse1) <= CURRENT_THRESH)
+
+    // Checking if ther is an overtemperature/short to ground condition
+    float vsenseh_efuse1 = io_loadswitch_isChannelEnabled(loadswitch->efuse1) / ADC_VOLTAGE_TO_CURRENT_A;
+    float vsenseh_efuse2 = io_loadswitch_isChannelEnabled(loadswitch->efuse2) / ADC_VOLTAGE_TO_CURRENT_A;
```

Comment:
am i tripping? are you dividing a bool by a float?

Reply:
surely you meant current right

Reply:
also is there a way we can test these guys in time for comp? :) (or have they been tested previously)